docs-ref-conceptual/install-azure-cli-yum.md

```diff
@@ -14,7 +14,7 @@ ms.devlang: azurecli
 # Install Azure CLI with yum
 
 For Linux distributions with  `yum` such as RHEL, Fedora, or CentOS, there's a package
-for the Azure CLI. This package has been tested with RHEL 7, Fedora 19 and higher, and CentOS 7.
+for the Azure CLI. This package has been tested with RHEL 7.7/8, Fedora 24 and higher, and CentOS 7/8.
```

Comment:
7.7 or 8 better?

Reply:
actually it should be `and`. will make the change.